src/api/flaskr/service/shifu/shifu_block_funcs.py

```
@@ -153,14 +154,25 @@ def save_shifu_block_list(
         save_block_ids = []
         save_block_models = []
         blocks_history = []
         is_changed = False
+        markdown_flow_content = ""
         for block in block_list:
             block_dto = convert_to_blockDTO(block)
+            try:
+                markdown_flow_content += "\n" + convert_block_to_mdflow(
+                    block_dto, variable_map
+                )
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)